src/renderers/shared/fiber/ReactFiberCommitWork.js

```diff
@@ -218,6 +218,10 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(
           }
         } else {
           if (isContainer) {
+            if (parentFiber.tag === HostPortal) {
+              // allows us to identify the portal container in other places
+              parent.__reactInternalIsPortalContainer = true;
```

Comment:
This is in shared renderer code but this expando is DOM specific. We shouldn't assume that all renderers are able to handle expandos. It should also be DEV only since it has perf considerations.